src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerCoordinatorViewModel.swift

```diff
@@ -78,7 +78,7 @@ class FileScannerCoordinatorViewModel: FileScannerProcessing {
 	init(
 		qrCodeDetector: QRCodeDetecting,
 		qrCodeParser: QRCodeParsable,
-		queue: DispatchQueue = .main
+		queue: DispatchQueue = DispatchQueue.global(qos: .background)
```

Comment:
```suggestion
		queue: DispatchQueue = DispatchQueue.global(qos: . userInitiated)
```
Feels like a better choice.
https://developer.apple.com/documentation/dispatch/dispatchqos/qosclass/userinitiated

Reply:
Done